src/model/wallet.ts

```diff
@@ -1294,6 +1305,6 @@ export const loadSeedPhraseAndMigrateIfNeeded = async (
   } catch (error) {
     logger.sentry('Error in loadSeedPhraseAndMigrateIfNeeded');
     captureException(error);
-    return null;
+    throw error;
```

Comment:
```suggestion
    throw null;
```

Reply:
we need to distinguish what error has occurred and throw it up

Reply:
why this was not needed before?

Reply:
We didn't need to determine what type of error occurred, we didn't show a recovery phrase for any of the errors. Now in case of an access error, we need not only to hide the phrase but also to show notifications with instructions, and that throw help to understand when we should show this instruction